views/newEvent.handlebars

```diff
@@ -1,112 +1,142 @@
-        <div class="container d-flex justify-content-center">
-            <form>
-
-                <div class="row" style="margin-bottom: 30px;">
-                    <div class="twelve columns" style="text-align: center;">
-                    <h2>Create an Event</h2>
-                    <p>Your first steps to creating your event begin with these few questions. You'll be on your way to securing new volunteers in no time!</p>
-                    </div>
+      <!-- Event information form -->
+      <div class="container d-flex justify-content-center">
+        <form>
+            <div class="row" style="margin-bottom: 30px;">
+                <div class="twelve columns" style="text-align: center;">
+                <h2>Create an Event</h2>
+                <p>Your first steps to creating your event begin with these few questions. You'll be on your way to securing new volunteers in no time!</p>
                 </div>
+            </div>
+        
+            <div class="form-row">
+              <div class="form-group col-md-12">
+                <label for="inputEvent">Event Name</label>
+                <input type="text" class="form-control" id="inputEventName">
+              </div>
+            </div>
+            <div class="form-group">
+              <label for="inputAddress">Address</label>
+              <input type="text" class="form-control" id="inputAddress" placeholder="1234 Main St">
+            </div>
+            <div class="form-group">
+              <label for="inputAddress2">Address 2</label>
+              <input type="text" class="form-control" id="inputAddress2" placeholder="Apartment, studio, or floor">
+            </div>
+            <div class="form-row">
+              <div class="form-group col-md-6">
+                <label for="inputCity">City</label>
+                <input type="text" class="form-control" id="inputCity">
+              </div>
+              <div class="form-group col-md-4">
```

Comment:
Lines 13-30 should match the tags in the newEvent.js file.

Reply:
Next step is to check the routes and make sure tags are appropriate